src/components/Loaders/taskDetailsShimmer/taskDetailsShimmer.module.scss

```diff
@@ -0,0 +1,102 @@
+@import '../../../styles/variables.scss';
+
+.taskCard {
+    border: 2px solid $white;
+    box-shadow: 0px 0px 10px 0 $light-gray;
+    padding: 10px;
+    border-radius: 8px;
+
+    min-width: 300px;
+    min-height: 110px;
+
+    margin-bottom: 2rem;
+
+    & > * {
+        border-radius: 8px;
+        margin: 10px;
+    }
+
+    .title {
+        min-height: 25px;
+        width: 60%;
+    }
+
+    .estimated {
+        min-height: 25px;
+        width: 40%;
+    }
+
+    .started {
+        min-height: 25px;
+        width: 20%;
+    }
+
+    .status {
+        min-width: 25px;
+        width: 20%;
+    }
```

Comment:
why is width being set twice here?

Reply:
1. One is setting the width
2. Another is to define the min-width while the screen resolution differs